sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/csv/CSVDataSource.scala

```diff
@@ -254,7 +256,7 @@ object TextInputCSVDataSource extends CSVDataSource {
         val header = makeSafeHeader(firstRow, caseSensitive, parsedOptions)
         val sampled: Dataset[String] = CSVUtils.sample(csv, parsedOptions)
         val tokenRDD = sampled.rdd.mapPartitions { iter =>
-          val filteredLines = CSVUtils.filterCommentAndEmpty(iter, parsedOptions)
+          val filteredLines = filterCommentAndEmpty(iter, parsedOptions)
```

Comment:
not a big deal but let's just use `CSVUtils...` usage just for consistency in this file.

Reply:
+1